animareflection/ui
# Breadcrumb Component

## Description

##### Task link: https://trello.com/c/Dn188wus/104-breadcrumb-component

Added `Breadcrumb` component.

## Test Steps

1. Ensure the component's structure, stories, and testing suite are robust.
2. Confirm that all tests are successful.
3. Check that the Breadcrumb displays correctly in both storybook and sample downstream applications.
4. Verify the use of demoPathname is suitable in the vite-react example instead of implementing routing dependencies.
5. Consider recipe styling for the address bar that may be useful in the future.
